docs/k8s-cli-installation.md

```diff
@@ -119,15 +119,6 @@ kubectl create namespace "${NAMESPACE}"
 
 ### Creating the Service Account
 
```

Comment:
Looks like we can remove this section header.

Reply:
Good catch, thanks.